src/main/java/net/earthcomputer/clientcommands/render/RenderQueue.java

```diff
@@ -0,0 +1,55 @@
+package net.earthcomputer.clientcommands.render;
+
+import net.minecraft.client.util.math.MatrixStack;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.function.Consumer;
+
+public class RenderQueue {
+
+    private static final HashMap<String, List<Consumer<MatrixStack>>> queue = new HashMap<>();
```

Comment:
The key to this map could now simply be an InjectLoc rather than String. And if you're doing that, may as well change it from a HashMap to an EnumMap.